docs/ddl-of-carbondata.md

```diff
@@ -750,10 +750,6 @@ Users can specify which columns to include and exclude for local dictionary gene
      ALTER TABLE test_db.carbon COMPACT 'SEGMENT_INDEX'
      ```
 
-     **NOTE:**
-
-     * Merge index is not supported on streaming table.
```

Comment:
Merge index is supported on the streaming table since 2.1. But the streaming segments don't merge index still.

Reply:
ok. done